src/Illuminate/Contracts/Validation/NotCompromisedVerifier.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Illuminate\Contracts\Validation;
+
+interface NotCompromisedVerifier
+{
+    /**
+     * Verify that the given value has not been compromised in data leaks.
+     *
+     * @param  string  $value
```

Comment:
I believe `string` is too strict here. Would you reckon opening it for more use cases?

Reply:
The `param` is not typed. So We can make this an union type later.